extensions/pom.xml

```diff
@@ -45,6 +45,7 @@
         <module>guacamole-auth-header</module>
         <module>guacamole-auth-jdbc</module>
         <module>guacamole-auth-json</module>
+        <module>guacamole-auth-nextcloud</module>
```

Comment:
This should be moved down below `ldap` to maintain alphabetic sorting.

Reply:
Oh sorry, I didn't notice that. I've fixed it.